Monika-After-Story/MonikaModDev
# folder-based hair and ACS sprites

#8596 

# Key Changes
* hair and ACS sprite objects now support folder-based organization, basically similar to clothes:
    * `h/<hair name>/front.png` or `h/<hair name>/0.png`
    * `a/<acs name>/0.png`
* sprite JSONs support the new format as well
   * this is based on a version - so version 4 and beyond must be folder-based
   * ver 3 sprites will continue to be supported but a warning will be logged to consider updating the version
* a simple bash script for renaming using git is provided, but a full-fledged spritepack updater will be added in #9729 

# Testing
* verify that all current sprites load correctly
* verify spritepacks with jsons **do not break** - testing of the new json props should be in #9729 for ease of testing
